cache/cache.go

```diff
@@ -191,14 +235,27 @@ func (c *Cache) Close() {
 func serveWatchEvents(ctx context.Context, watchCtx *watchCtx) {
 	backoff := watchCtx.backoffStart
 	for {
+		var startRev int64
+		if kv, ok := watchCtx.cache.watcher.(clientv3.KV); ok {
+			if resp, err := kv.Get(ctx, watchCtx.cache.prefix, clientv3.WithPrefix()); err == nil {
+				events := make([]*clientv3.Event, 0, len(resp.Kvs))
+				for _, kv := range resp.Kvs {
+					events = append(events, &clientv3.Event{
+						Type: clientv3.EventTypePut,
+						Kv:   kv,
+					})
+				}
+				watchCtx.cache.demux.Broadcast(events)
```

Comment:
What is this? this is not how eventually consistent cache works. You don't convert state into diff.